core/math/src/main/java/org/arakhne/afc/math/MathUtil.java

```diff
@@ -185,7 +187,7 @@ public static boolean isEpsilonEqual(double v1, double v2) {
 	@Pure
 	public static boolean isEpsilonEqual(double v1, double v2, double epsilon) {
 		if (Double.isInfinite(v1)) {
-			return Double.isInfinite(v2) && Math.signum(v1) == Math.signum(v2);
+			return Double.isInfinite(v2) && areFloatsEqual(Math.signum(v1), Math.signum(v2));
```

Comment:
This is a waste of time. Indeed signum function replies -1, 0, or 1. We can use the == operator.